### PR TITLE
use `torch.clamp`

### PR DESCRIPTION
**Pull request by vincentzed**
## Motivation
Motivated by the note in code, we should use `torch.clamp` cc @merrymercy  if you could please look.

https://pytorch.org/docs/stable/generated/torch.clamp.html
## Modifications

use `torch.clamp` for val
> The `torch.minimum` function calculates the element-wise minimum between the seq_lens tensor and a scalar tensor `self.sliding_window_size + 1`. Now, no value in `paged_kernel_lens_tmp` exceeds `self.sliding_window_size + 1`. Values in `seq_lens` that are already less than or equal to this limit remain unchanged.

## Checklist

- [x] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [x] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

**Comment by vincentzed** [comment]
@zhaochenyang20 If you can help take a look ⚡
